Modules/_testcapimodule.c

```diff
@@ -4051,6 +4051,86 @@ raise_SIGINT_then_send_None(PyObject *self, PyObject *args)
 }
 
 
+static PyObject *
+test_pyobject_fastcall(PyObject *self, PyObject *args)
```

Comment:
I would remove the `test_` prefix (and may be even `pyobject_`). Functions with the `test_` prefix usually do all testing internally, they don't return a value for checking in Python test.